cookies/operations.py

```diff
@@ -462,8 +428,19 @@ def generate_collection_coauthor_graph(collection,
 
     return graph
 
-@mock.patch('requests.get')
-def concept_search(text, mock_get):
+
+def ping_remote_resource(path):
```

Comment:
It looks like you just copied this function from [views.py](https://github.com/diging/amphora/blob/972c16dcc4b7604ea77867fc4e4de19d07db8f8c/cookies/views.py#L54). We definitely don't want the same function defined in two places. The function in views.py is a private function, so if you need to use this you could consider eliminating the one in views.py; just be sure to correct any code in that module that depends on the old function to use this new one in operations.py.